Language/Expressions/Constants/as_type_A01_t02.dart

```diff
@@ -74,7 +73,6 @@ main() {
 
   const f1 = foo as void Function();
   const f2 = as as void Function();
-  const f3 = bar<int> as int Function(int);
 
```

Comment:
Let's keep everything about `bar`: The declaration, this const, and the test in line 91.

There is nothing wrong with the test itself, we just can't express it in the given syntactic form. So we'll have to use this in order to make line 77 work, and the rest should then work just fine:

```
const f3 = (bar<int>) as int Function(int);
```

The term `bar<int>` in line 91 should be fine because it is followed by `,`.

Reply:
Thank you. Updated